includes/html/graphs/bill/historictransfer.inc.php

```diff
@@ -4,14 +4,15 @@
 use Amenadiel\JpGraph\Plot\BarPlot;
 use Amenadiel\JpGraph\Plot\GroupBarPlot;
 use Amenadiel\JpGraph\Plot\LinePlot;
+use LibreNMS\Billing;
 use LibreNMS\Util\Number;
 
 if (is_numeric($vars['bill_hist_id'])) {
-    $graph_data = getBillingBandwidthGraphData($vars['id'], $vars['bill_hist_id'], null, null, $vars['imgtype']);
+    $graph_data = Billing::getBandwidthGraphData($vars['id'], $vars['bill_hist_id'], null, null, $vars['imgtype']);
 } elseif (is_numeric($vars['from'])) {
-    $graph_data = getBillingBandwidthGraphData($vars['id'], null, $vars['from'], $vars['to'], $vars['imgtype']);
+    $graph_data = Billing::getBandwidthGraphData($vars['id'], null, $vars['from'], $vars['to'], $vars['imgtype']);
 } else {
-    $graph_data = getHistoricTransferGraphData($vars['id']);
+    Billing::getHistoricTransferGraphData($vars['id']);
```

Comment:
Same here?

Reply:
omg

Reply:
Must have been the stupid refactoring tool that messed up.